src/main/java/seedu/address/logic/commands/list/ListApplicationCommand.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.logic.commands.list;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.commands.util.CommandUtil.getCommandResult;
+import static seedu.address.model.FilterableItemList.PREDICATE_SHOW_ALL_ITEMS;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.Model;
+import seedu.address.ui.tabs.TabName;
+
+/**
+ * todo javadocs
+ */
+public class ListApplicationCommand extends ListCommand {
+
+    public static final String COMMAND_WORD = "list";
```

Comment:
Is this redundant given that `ListApplicationCommand` can use `ListCommand`'s `COMMAND_WORD`? This is with regards to how all the other commands exhibit the same pattern in that they use the parent's `COMMAND_WORD`.

Reply:
Ah, yes that is true, will make the relevant changes.